docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -688,52 +975,58 @@ Ensure you have the following files in the current working directory.
 
 Note that the Terraform code will deploy the resources to **AWS**. 
 
-We recommend you explore all Terraform files. Below is a high-level overview of each file:
+We recommend that you explore all Terraform files. Below is a high-level overview of each file.
 <br />
 
 -  **profile.tf** - contains the configuration for the `spectrocloud_cluster_profile` resource. Review the core infrastructure layers that make up the `spectrocloud_cluster_profile` resource. 
 
 
 
--  **cluster.tf** - contains the configuration for the `spectrocloud_cluster_aws` resource. The cluster resource depends upon the `spectrocloud_cluster_profile` resource. 
+-  **cluster.tf** - contains the configuration for the `spectrocloud_cluster_aws` resource. The cluster resource depends on the `spectrocloud_cluster_profile` resource. 
 
 
 
 - **data.tf** - contains the configuration for the resources to retrieve data from Palette dynamically. The table below lists the pack details required for each pack layer in order to deploy the `spectrocloud_cluster_profile` resource. 
 
   |**Pack Type**|**Registry**|**Pack Name**|**Tag**| **Version** |
-  |---|---|---|---|
-  |OS|Public Repo|`ubuntu-aws`|`LTS__20.4.x`| `20.04`|
-  |Kubernetes|Public Repo|`kubernetes`|`1.24.x`| `1.24.10` |
-  |Network|Public Repo|`cni-calico`|`3.25.x`|`3.25.0`|
-  |Storage|Public Repo|`csi-aws-ebs`|`1.16.x`|`1.16.0`|
-
-  Note that using this Terraform code will deploy the resources to AWS. To deploy your resource to Azure or Google Cloud, use the layer details outlined in [Cloud Service Provider Configurations] (https://github.com/spectrocloud/tutorials/tree/main/terraform/pack-tf/README.md#cloud-service-provider-configurations). 
+  |---|---|---|---|---|
+  |OS|Public Repo|`ubuntu-aws`|`LTS__22.4.x`| `22.04`|
+  |Kubernetes|Public Repo|`kubernetes`|`1.28.x`| `1.28.2` |
+  |Network|Public Repo|`cni-calico`|`3.26.x`|`3.26.1`|
+  |Storage|Public Repo|`csi-aws-ebs`|`1.22.x`|`1.22.0`|
+  |Authentication|Public Repo|`spectro-proxy`|`1.4.x`|`1.4.1`|
```

Comment:
Carolina, let's take out the Spectro Proxy pack from the Terraform code. If will break if the users tries to use the kubeconfig file with kubectl. The reason it breaks is because the k8s layer YAML needs the certSAN config parameter. As added in the UI workflow. So let's remove it here and from the code.